docs/src/enterprise/getstarted/migrate-from-fluffy.md

```diff
@@ -0,0 +1,316 @@
+              logout_endpoint_query_parameters:
+                - returnTo
+                - http://localhost:8000/oidc/login
+        ```
+
+
+!!! note "LDAP"
+   
+    === "lakeFS + Fluffy (old)"
+        
+        ```yaml
+        # fluffy.yaml
+        auth:
+          post_login_redirect_url: /
+          ldap: 
+            server_endpoint: 'ldaps://ldap.company.com:636'
+            bind_dn: uid=<bind-user-name>,ou=<some-ou>,o=<org-id>,dc=<company>,dc=com
+            bind_password: '<ldap pwd>'
+            username_attribute: uid
+            user_base_dn: ou=<some-ou>,o=<org-id>,dc=<company>,dc=com
+            user_filter: (objectClass=inetOrgPerson)
+            connection_timeout_seconds: 15
+            request_timeout_seconds: 7
+	    ```
+        ```yaml
+    	# lakefs.yaml
+        auth:
+          remote_authenticator:
+            enabled: true
+            endpoint: http://<Fluffy URL>:<Fluffy http port>/api/v1/ldap/login
+            default_user_group: "Developers" # Value needs to correspond with an existing group in lakeFS
+          ui_config:
+            logout_url: /logout
+            login_cookie_names:
+            - internal_auth_session
+        ```
+    
+    === "lakeFS Enterprise (new)"
+        
+        ```yaml
+        # lakefs.yaml
+        auth:
+          ui_config:
+            logout_url: /logout
+            login_cookie_names:
+              - internal_auth_session
+          providers:
+            ldap:
+              server_endpoint: ldaps://ldap.jumpcloud.com:636
+              bind_dn: uid=bind_app,ou=Users,o=63eb8e248d5156c875be4124,dc=jumpcloud,dc=com
+              bind_password: g4udFrCC8FXyU@c
+              username_attribute: uid 
+              user_base_dn: ou=Users,o=63eb8e248d5156c875be4124,dc=jumpcloud,dc=com 
+              user_filter: (objectClass=inetOrgPerson)
+              connection_timeout_seconds: 15
+              request_timeout_seconds: 7
+              default_user_group: "admins" // This field moved here!
+        ```
+
+!!! note "AWS IAM"
+   
+    === "lakeFS + Fluffy (old)"
+        
+        ```yaml
+        # fluffy.yaml
+        auth:
+          serve_listen_address: "localhost:9000"
+          external:
+            aws_auth:
+              enabled: true
+              required_headers:
+                X-LakeFS-Server-ID: "localhost"
+        ```
+        ```yaml
+		# lakefs.yaml
+        auth:
+          authentication_api:
+            endpoint: http://localhost:9001/api/v1
+            external_principals_enabled: true
+          api:
+           endpoint: http://localhost:9000/api/v1
+        ```
+    
+    === "lakeFS Enterprise (new)"
+        
+        ```yaml
+        # lakefs.yaml
+        auth:
+          external_aws_auth:
+            enabled: true
+            required_headers:
+              X-LakeFS-Server-ID: "localhost"
+
+        ```
+
+!!! note "STS auth"
+    The structure of STS authentication remains the same.
```

Comment:
Just add that to OIDC section to make sure there's no confusion here as a BTW. 
(OIDC + OIDC STS auth)  or something